test/test_cuda.py

```diff
@@ -294,20 +294,100 @@ def test_memory_allocation(self):
 
     def test_get_alloc_free_events(self):
         gc.collect()
+        torch.cuda.set_memory_tracker()
         size = 1
         mem = torch.cuda.caching_allocator_alloc(size)
         current_device = torch.cuda.current_device()
         events = torch.cuda.get_alloc_free_events()[current_device]
         num_events = len(events)
         ptr = events[-1].ptr
-        self.assertTrue(events[-1].size == 1)
+        self.assertTrue(events[-1].size == size)
 
         torch.cuda.caching_allocator_delete(mem)
         events = torch.cuda.get_alloc_free_events()[current_device]
         self.assertTrue(len(events) == (num_events + 1))
         self.assertEqual(events[-1].ptr, ptr)
         self.assertTrue(events[-1].size < 0)
 
+    def test_set_disable_memory_tracker(self):
+        gc.collect()
+        torch.cuda.set_memory_tracker()
+        size, size2 = 1, 2
+        current_device = torch.cuda.current_device()
+        mem = torch.cuda.caching_allocator_alloc(size)
+        events = torch.cuda.get_alloc_free_events()[current_device]
+        self.assertTrue(events[-1].size == size)
+
+        torch.cuda.disable_memory_tracker()
```

Comment:
curious, why set/disable instead of enable/disable?

Reply:
Done. Thank you.